packages/elements/src/components/file/file.component.ts

```diff
@@ -134,15 +134,17 @@ export class FileComponent extends RealTimeElement {
 
   private renderMutantDots(mutants: MutantModel[] | undefined) {
     return mutants?.length
-      ? mutants.map(
-          (mutant) =>
-            svg`<svg mutant-id="${mutant.id}" class="mutant-dot ${
-              this.selectedMutant?.id === mutant.id ? 'selected' : mutant.status
-            }" height="10" width="12">
-          <title>${title(mutant)}</title>
-          <circle cx="5" cy="5" r="5" />
-          </svg>`,
-        )
+      ? mutants.map((mutant) => {
+          const selected = this.selectedMutant?.id === mutant.id;
+          const selectedClass = selected ? 'selected' : '';
+
+          return svg`
+            <svg mutant-id="${mutant.id}" class="mutant-dot ${selectedClass} ${mutant.status}" height="10" width="12">
+              <title>${title(mutant)}</title>
+              
+              ${selected ? svg`<polygon class="mutant-dot-polygon" points="5,10 0,0 10,0"  />` : svg`<circle cx="5" cy="5" r="5" color="red" />`}
```

Comment:
```suggestion
              ${selected ? svg`<polygon class="mutant-dot-polygon" points="5,10 0,0 10,0"  />` : svg`<circle cx="5" cy="5" r="5" />`}
```

We don't want all mutant dots to be red